### ISSUE TITLE
Add an e2e test for the Delete action

### ISSUE THREAD

**Issue by EstefiCamba** [Cambalab/vue-admin#54]

**Comment by oskar-kupski-elpassion** [comment]
I could give it a try

**Comment by sgobotta** [comment]
> I could give it a try

Alright, comment here if you need any hints.

**Comment by oskar-kupski-elpassion** [comment]
What value can I pass as `resourceName` to make valid dispatch? I tried with 'magazines', so that generated action type in onDelete() method is 'magazines/destroy'

**Comment by sgobotta** [comment]
> What value can I pass as `resourceName` to make valid dispatch? I tried with 'magazines', so that generated action type in onDelete() method is 'magazines/destroy'

Yeah, hardcoding `magazines` is ok, or maybe you would like to define a `const` with the `magazines` string if you're using it repeatedly across tests.

Here's an [example](https://github.com/Cambalab/vue-admin/blob/develop/tests/unit/specs/admin/admin.spec.js#L171) but using constants from the vue-admin core.

**Comment by oskar-kupski-elpassion** [comment]
hmm, when I tried hardcoding it with `magazines` I got an error "unknown action type: magazines/destroy"

**Comment by sgobotta** [comment]
That makes sense if you didn't initialise a mocked store, here's an [example](https://github.com/Cambalab/vue-admin/blob/master/tests/unit/specs/resource.spec.js#L18) of that initialisartion.

In your test you'll have to define a fixture in the `test/unit/fixtures` directory for the delete action spec that declares the storeMethods, like [this one](https://github.com/Cambalab/vue-admin/blob/master/tests/unit/fixtures/resource/magazines.js#L21)

**Comment by sgobotta** [comment]
@oskar-kupski-elpassion are you still working on this, do you need any help? If you gave up on this issue please let me know to un-assign you. Thanks!

**Comment by oskar-kupski-elpassion** [comment]
Please unassign me
